repos.d/gentoo/liguros.yaml

```diff
@@ -1,39 +1,37 @@
 ###########################################################################
 # Liguros
 ###########################################################################
-- name: liguros_20_7
+- name: liguros_21_7
```

Comment:
Since sources do not refer to a fixed version, please use corresponding generic `name`/`desc`, e.g. `liguros_develop`/`LiGurOS develop`